SilMock/Google/Service/Directory/VerificationCodesResource.php

```diff
@@ -0,0 +1,135 @@
+<?php
+namespace SilMock\Google\Service\Directory;
+
+use SilMock\DataStore\Sqlite\SqliteUtils;
+use SilMock\Google\Service\Directory;
+
+class VerificationCodesResource
+{
+
+    private $_dbFile;  // string for the path (with file name) for the Sqlite database
+    private $_dataType = 'directory';  // string to put in the 'type' field in the database
+    private $_dataClass = 'verification_codes'; // string to put in the 'class' field in the database
+
+    public function __construct($dbFile = null)
+    {
+        $this->_dbFile = $dbFile;
+    }
+
+    /**
+     * invalidate all the verification codes for a particular user.
+     *
+     * @param string|int $userKey The email or immutable Id of the user
+     * @return true|null depending on if an alias was deleted
+     * @throws \Exception with code 201407101645
```

Comment:
Why that particular exception code? Is that what Google will/might return?

Reply:
See SilMock/Google/Service/Directory/UsersAliasesResource.php

Reply:
Ahh, yes let's not use that code, then. Looks like it was just a (more or less) arbitrary code added when that other function was written: 201407101645, from 2014/07/10 16:45

When using that style, the purpose was to use a globally unique error code. I'd recommend we leave off the error code instead, though if there's a reason to have one, let's replace this with something like the current timestamp (which probably hasn't been used elsewhere in our code).

Reply:
done.